### PR TITLE
[GITHUB-6] Fix for new nodejs source repository

### PR DESCRIPTION
**Pull request by wojtek-oledzki**
The ansible apt modules works only with python >= 2.7.9 with SNI support
enabled.

### CODE REVIEW

File: tasks/build.yml [hunk 1]
```diff
@@ -1,16 +1,14 @@
...
-
+  # in order to use `apt_key` with nodejs repos you need python >= 2.7.9
+  # with SNI support
 - name: Add NodeSource deb repository
   become: yes
-  apt_repository:
-    repo: "deb https://deb.nodesource.com/node_{{ nodejs.version }}.x {{ ansible_distribution_release }} main"
-    state: present
+  shell: "{{ item }}"
+  with_items:
```
Comment (lobsterdore) [inline tasks/build.yml]: I think you should use the validate_certs option of the apt_key and apt_repository modules if possible.

File: tasks/main.yml [hunk 2]
```diff
@@ -6,8 +6,8 @@
     - build
 
 - name: configure noop
```
Comment (lobsterdore) [inline tasks/main.yml:9]: Can we get rid of the dummy configure?

File: tasks/build.yml (hunk 1 above)
Comment (wojtek-oledzki) [reply tasks/build.yml]: error with `validate_certs: no` :(

```
TASK [sansible.nodejs : Import the NodeSource GPG key into apt] ****************
"changed": false, "failed": true, "msg": "Failed to download key at https://deb.nodesource.com/gpgkey/nodesource.gpg.key: Request failed: <urlopen error [Errno 1] _ssl.c:510: error:14077410:SSL routines:SSL23_GET_SERVER_HELLO:sslv3 alert handshake failure>
```

Comment (lobsterdore) [reply tasks/build.yml]: Perhaps we should just recommend a newer version of Python? It's easy enough to install on most systems, there is a ppa available for Ubuntu 14.

File: tasks/main.yml (hunk 2 above)
Comment (wojtek-oledzki) [reply tasks/main.yml:9]: not as long as we support 1.9.

**Comment by lobsterdore** [comment]
LGTM :+1:

**Pull request merged**